public/app/features/explore/ContentOutline/ContentOutlineContext.tsx

```diff
@@ -1,6 +1,8 @@
 import { uniqueId } from 'lodash';
 import { useState, useContext, createContext, ReactNode, useCallback, useRef, useEffect } from 'react';
 
+import { MakeOptional } from '../../trails/shared';
```

Comment:
It feels weird to import something from trails :D it could be move to grafana/data but instead I think we can just use `import { SetOptional } from 'type-fest';` which we already import. Should do the trick.

Reply:
Fixed!